__tests__/app.test.js

```diff
@@ -42,3 +42,34 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  test("returns an array of reviews sorted by the creation date of the reviews (desc) and containing a number of comments column", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+
+        expect(Array.isArray(reviews)).toBe(true);
+        expect(reviews.length).toBeGreaterThan(0);
+        expect(reviews[0] && typeof reviews[0] === "object").toBe(true);
+        expect(reviews).toBeSortedBy("created_at", {
+          descending: true,
+        });
+
+        reviews.forEach((review) => {
+          expect(review).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            count: expect.any(String),
```

Comment:
This property you're testing for on line 70 should be comment_count not just count.